template/info_label.txt

```diff
@@ -17,3 +17,5 @@
 13, spinal cord centerline, PAM50_centerline.nii.gz
 14, point-wise spinal level labels, PAM50_spinal_midpoint.nii.gz
 15, spinal cord mask with discrete spinal levels, PAM50_spinal_levels.nii.gz
+16, spinal dorsal nerve rootlets with discrete spinal levels, PAM50_rootlets.nii.gz
+17, spinal dorsal nerve rootlets center of mass with discrete spinal levels, PAM50_rootlets_mid_center.nii.gz
```

Comment:
Should we project onto `PAM50_centerline.nii.gz` rather than `PAM50_cord.nii.gz` so these two files overlap?

![image](https://github.com/user-attachments/assets/ff2cc544-1752-4e2b-b87c-15c8ac898b50)

![image](https://github.com/user-attachments/assets/13efa573-cdba-4545-a091-af11cb7f8a19)

Reply:
yes sounds good, just modified here: https://github.com/spinalcordtoolbox/PAM50/pull/31/commits/e3ae8f9c310564ae1a2e315d91bc6972e83a4f34

I update the PR body too